src/main/kotlin/no/nav/personbruker/dittnav/metrics/periodic/reporter/metrics/kafka/kafkaCountingUtil.kt

```diff
@@ -17,7 +17,7 @@ import java.time.Duration
 import java.time.Instant
 import java.time.temporal.ChronoUnit
 
-private val log = LoggerFactory.getLogger(KafkaEventCounterService::class.java)
+private val log = LoggerFactory.getLogger(object{}::class.java.`package`.name)
```

Comment:
Bør vi heretter gjøre det slik ved logger-instansiering utenfor klasser?

Reply:
Jeg har ikke gjort veldig grundig research rundt dette, men det virket som en grei måte å gjøre det på. Man slipper bla å peke på en annen klasse som egentlig dette ikke hører til, eller å måtte finne på en konvensjon for hvordan man skal navngi logger for filer uten klasser.

Reply:
Loggeren får i praksis samme navn som pakken filen ligger i, så man kan eventuelt kun kalle loggeren for pakkenavnet ved å skrive det inn manuelt.